dist/tools/tunslip/tunslip6.c

```diff
@@ -650,9 +650,10 @@ devopen(const char *dev, int flags)
 {
     char t[1024];
     strcpy(t, "/dev/");
-    /* cppcheck-suppress bufferAccessOutOfBounds
-     * reason: seems to be a cppcheck bug */
-    strncat(t, dev, sizeof(t) - 5);
+    /* need to consider `\0` terminator and strlen("dev") = 5, hence 6
+     * cppcheck-suppress bufferAccessOutOfBounds
+     * (reason: seems to be a bug in cppcheck 1.7x) */
+    strncat(t, dev, sizeof(t) - 6);
```

Comment:
Also: `cppcheck` still isn't happy with this line: https://ci.riot-os.org/RIOT-OS/RIOT/10010/acdbd77214e4aefa76f539075643591cc64d8bc2/output.html

Reply:
strange, I simply squashed and rebased, why was murdock happy before?